app/src/app/index.jsx

```diff
@@ -162,6 +162,7 @@ const MarketplaceRouter = () => (process.env.COMMUNITY_PRODUCTS ? [
     <Route exact path={links.marketplace.createProduct} component={CreateProductAuth2} key="CreateProduct" />,
     <Route exact path={formatPath(marketplace.products, ':id', 'purchase2')} component={ProductPurchasePage2} key="ProductPurchasePage2" />,
     <Route exact path={formatPath(marketplace.products, ':id', 'publish2')} component={ProductPublishPage2} key="ProductPublishPage2" />,
+    <Route exact path={formatPath(marketplace.products, ':id', 'streamPreview', ':streamId')} component={StreamPreviewPage} key="StreamPreview" />,
```

Comment:
good catch, I suppose this is necessary for the next MP deployment?

Reply:
This is also included in https://github.com/streamr-dev/streamr-platform/pull/755 and it's needed only when we enable COMMUNITY_PRODUCTS. No need to hurry yet since the route is present in the "old" version.